themes/bootstrap3/templates/search/advanced/checkbox-filters.phtml

```diff
@@ -9,4 +9,4 @@
       </div>
     <? endforeach; ?>
   </div>
-<?endif;?>
+<? endif; ?>
```

Comment:
This looks like a style fix that should probably be merged to master separately.